recipes/default.rb

```diff
@@ -68,9 +68,9 @@
 remote_file "#{Chef::Config[:file_cache_path]}/install.sig" do
   source              node.default[:inspector][:gpg_signature_url]
   use_conditional_get true
-  mode                0440
+  mode                0o440
```

Comment:
What's this change?

Reply:
Does the 'o' mean octal? Here's what's in the document https://docs.chef.io/resource_directory.html

> UNIX- and Linux-based systems: A quoted 3-5 character string that defines the octal mode that is passed to chmod. For example: '755', '0755', or 00755. If the value is specified as a quoted string, it works exactly as if the chmod command was passed. If the value is specified as an integer, prepend a zero (0) to the value to ensure that it is interpreted as an octal number. For example, to assign read, write, and execute rights for all users, use '0777' or '777'; for the same rights, plus the sticky bit, use 01777 or '1777'.

Reply:
yes it stands for octal

Reply:
I can remove it if you want just a stylistic change, I spend too much time writing ruby so I spot these kinds of things but the previous is valid.

Reply:
No, IMO explicit is better. Always.